src/utils/string-test.js

```diff
@@ -162,3 +163,19 @@ describe('swgTransactionId', () => {
     }
   });
 });
+
+describe('hash', async () => {
+  it('should create 128 character hexadecimal hashes', async () => {
+    let result = null;
+    await hash('string1').then((h) => (result = h));
+    expect(result.length).to.equal(128);
```

Comment:
Please check against an actual hash value instead of length

Reply:
I'm not sure I agree in this case, we can discuss over chat if you'd like.  I did improve the tests some.